task/04-arrays-tasks.js

```diff
@@ -38,7 +38,7 @@ function findElement(arr, value) {
  *    5 => [ 1, 3, 5, 7, 9 ]
  */
 function generateOdds(len) {
-   throw new Error('Not implemented');
+   return Array(len).fill(0).map((x, i) => x = 2 * i + 1);
```

Comment:
Use `new Array` instead of `Array`. It is a potential bug. Why? If you see MDN, you will not find anything about `Array` function.

Reply:
Use more variables.

Reply:
Use more meaningful names for variables. For example: x -> value, i -> index.